osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -23,7 +23,7 @@ public Aim(Mod[] mods, bool withSliders)
 
         private double currentStrain;
 
-        private double skillMultiplier => 23.55;
+        private double skillMultiplier => 23.55 * 1.06;
```

Comment:
Why not multiply it in already? I'm not against small %diffs.

Reply:
fixed